composer/core/state.py

```diff
@@ -10,6 +10,7 @@
 from typing import TYPE_CHECKING, Any, Callable, Dict, Iterable, Optional, Sequence, Union, cast
 
 import torch
+import torch.nn
```

Comment:
why is there an import change but no changes in the file?

Reply:
Pyright in vscode doesn't like implicit imports of submodules for some reason...

Reply:
(This is coming along for the ride...)